package.json

```diff
@@ -87,6 +87,9 @@
     "tools",
     "zendesk"
   ],
+  "engines": {
+    "node": ">=18.0.0"
```

Comment:
Does a specific package require min 18?

Reply:
Global `fetch` used by the `netlify-bandwidth` command (`node-fetch` CJS package no longer needed).